desktop-exporter/otlp_payload_processor.go

```diff
@@ -8,26 +8,54 @@ import (
 	"go.opentelemetry.io/collector/pdata/ptrace"
 )
 
+type ResourceData struct {
+	Attributes             map[string]interface{}
+	DroppedAttributesCount uint32
+}
+
+type ScopeData struct {
+	Name                   string
+	Version                string
+	Attributes             map[string]interface{}
+	DroppedAttributesCount uint32
+}
+
 type SpanData struct {
-	TraceID      string
+	TraceID    string
+	TraceState string
+
 	SpanId       string
```

Comment:
nit: `SpanId` should be written like `TraceID`, like `SpanID`.

I think that the go linter will call things like this out, so we should hook that up soon.

Reply:
Oh dear. Yes it definitely should and I have no idea how I haven't noticed it yet. Good call on the linter.